tests/templates/test_subroutines/test_trotter.py

```diff
@@ -162,6 +163,47 @@
 }
 
 
+def _generate_simple_decomp(coeffs, ops, time, order, n):
+    """Given coeffs, ops and a time argument in a given framework, generate the
+    Trotter product for order and number of trotter steps."""
+    decomp = []
+    if order == 1:
+        decomp.extend(qml.exp(op, coeff * (time / n) * 1j) for coeff, op in zip(coeffs, ops))
+
+    coeffs_ops = zip(coeffs, ops)
+
+    if get_interface(coeffs) == "torch":
+        import torch
+
+        coeffs_ops_reversed = zip(torch.flip(coeffs, dims=(0,)), ops[-1::-1])
+    else:
+        coeffs_ops_reversed = zip(coeffs[-1::-1], ops[-1::-1])
```

Comment:
`coeffs[::-1]` does not do the same?

Reply:
Im not sure, I have always used `[-1::-1]`.

Reply:
[non-blocking] might check and verify.

Reply:
replaced 👍🏼